translations/combat.tex/pl.po

```diff
@@ -71,25 +81,33 @@ msgid ""
 "Backlines and two Frontlines on opposite ends, and a middle row.  Follow "
 "these steps when Combat begins against \\textbf{Neutral Units}:"
 msgstr ""
+"Walka odbywa się na planszy Walki 4 x 5, którą podzielić można na dwie"
```

Comment:
W polskiej wersji Księgi Zasad są to Pierwsza i Druga Linia i może lepiej trzymać się tego? tu i w kolejnych użyciach